docs/versioning/how-to-view-versions.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: default
+nav_order: 5
+title: How to view versions  
+has_children: false
+parent: Versioning
+---
+# How to view versions
+
+
+## Viewing version info in the Registry
+
+
+### Step 1. Navigate to Versioning tab in the item record in the Registry
+
+- Ensure you're logged in.
+- Navigate to the Registry.
+- Navigate to the item record.
+- Click on the "VERSIONING" tab as shown below.
+
+|                         Versioning Tab                                                                 |
+| :----------------------------------------------------------------------------------------------------: |
+| <img src="../assets/images/versioning/versioning-registry-versioning-tab.jpg" alt="drawing" width="600"/> |
+
+
+You will be able to view the version history including the current and previous versions.
+
+|               Updated versioning details                                                    |
+| :----------------------------------------------------------------------------------------------------: |
+| <img src="../assets/images/versioning/versioning-registry-create-version-result.png" alt="drawing" width="600"/> |
+
+You can click on the icon with 3 dots in the versioning timeline to expand to display additional versions.
+
+<img src="../assets/images/versioning/versioning-registry-view-expand.png" alt="drawing" width="400"/>
+
+Sorting is available in the drop down menu.
+
+<img src="../assets/images/versioning/versioning-registry-view-sort.png" alt="drawing" width="400"/>
+
```

Comment:
Image width 👁️

Reply:
There are smaller images (hard to read) with a width of 400, maybe they should be all 600px wide